src/main/java/com/project/validation/MandatoryValueValidator.java

```diff
@@ -0,0 +1,11 @@
+package com.project.validation;
+
+import com.project.exception.ValidationException;
+
+public class MandatoryValueValidator extends Validator<String> {
+    @Override
+    public void validate(String attribute, String message) {
+        if(attribute == null || attribute.trim().isEmpty())
```

Comment:
You could use apache commons lib here `StringUtils.isEmpty()` checks whether `String` contains text and is `null` safe.

You can read more about it here:
https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html

Reply:
changed the `if` statement condition to `StringUtils.isEmpty()` method